services/movie.py

```diff
@@ -0,0 +1,34 @@
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: None = None,
+        actors_ids: None = None
+) -> None:
+    queryset = Movie.objects.all()
+    if genres_ids:
+        queryset = queryset.filter(genres__id__in=genres_ids).distinct()
+    if actors_ids:
+        queryset = queryset.filter(actors__id__in=actors_ids).distinct()
+    return queryset
+
+
+def get_movie_by_id(movie_id: id) -> id:

```

Comment:
The parameter `movie_id` should be of type `int` instead of `id`. Also, the return type should be `Movie`, not `id`. Update the function signature to `def get_movie_by_id(movie_id: int) -> Movie:`.